### DISCUSSION TITLE
LDAP AD Group Mappings Failing

### DISCUSSION THREAD

**Discussion by bernardburns** [apache/airflow#22205] [Q&A]
Hello all, we are encountering an issue authenticating using LDAP in Airflow 2.2.3, leveraging Flask Appbuilder 3.4.1.  In webserver_config.py, the mappings of the AD groups to the Airflow roles in AUTH_ROLES_MAPPING are being ignored. The AD account is being assigned to the default role specified by AUTH_USER_REGISTRATION_ROLE at time of logon, regardless of the mapping. Has anyone else encountered this or have an idea on what’s causing it?  Thank you!

**Comment by satish-dops94** [comment]
were you able to resolve this problem @bernardburns ? I am also stuck at the same issue and not able to figure out the problem
